css-two-column-layout/styles.css

```diff
@@ -0,0 +1,73 @@
+body {
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.column-full {
+  flex-basis: 700px;
```

Comment:
I would avoid using `px` for `flex-basis`, this will start causing you trouble when you want to start making things responsive. I would recommend using percents.